agenta-backend/agenta_backend/models/db_models.py

```diff
@@ -1,303 +1,470 @@
+    )
+
+
+class EvaluatorConfigDB(Base):
+    __tablename__ = "evaluators_configs"
+
+    id = Column(
+        UUID(as_uuid=True),
+        primary_key=True,
+        default=uuid.uuid7,
+        unique=True,
+        nullable=False,
+    )
+    evaluation_id = Column(UUID(as_uuid=True), ForeignKey("evaluations.id"))
+    evaluation = relationship("EvaluationDB", back_populates="evaluator_configs")
+    evaluation_scenario_id = Column(
+        UUID(as_uuid=True), ForeignKey("evaluation_scenarios.id")
+    )
+    evaluation_scenario = relationship(
+        "EvaluationScenarioDB", back_populates="evaluator_configs"
+    )
+    app_id = Column(UUID(as_uuid=True), ForeignKey("app_db.id"))
+    app = relationship("AppDB")
+    user_id = Column(UUID(as_uuid=True), ForeignKey("users.id"))
+    user = relationship("UserDB")
+    name = Column(String)
+    evaluator_key = Column(String)
+    settings_values = Column(JSONB, default=dict)
+    created_at = Column(
+        DateTime(timezone=True), default=lambda: datetime.now(timezone.utc)
+    )
+    updated_at = Column(
+        DateTime(timezone=True), default=lambda: datetime.now(timezone.utc)
+    )
+
+
+class HumanEvaluationDB(Base):
+    __tablename__ = "human_evaluations"
+
+    id = Column(
+        UUID(as_uuid=True),
+        primary_key=True,
+        default=uuid.uuid7,
+        unique=True,
+        nullable=False,
+    )
+    app_id = Column(UUID(as_uuid=True), ForeignKey("app_db.id"))
+    app = relationship("AppDB")
+    user_id = Column(UUID(as_uuid=True), ForeignKey("users.id"))
+    user = relationship("UserDB")
+    status = Column(String)
+    evaluation_type = Column(String)
+    variant_id = Column(UUID(as_uuid=True), ForeignKey("app_variants.id"))
+    variant = relationship("AppVariantDB")
+    variant_revision_id = Column(
+        UUID(as_uuid=True), ForeignKey("app_variant_revisions.id")
+    )
+    variant_revision = relationship("AppVariantRevisionsDB")
+    testset_id = Column(UUID(as_uuid=True), ForeignKey("testsets.id"))
+    testset = relationship("TestSetDB")
+    created_at = Column(
+        DateTime(timezone=True), default=lambda: datetime.now(timezone.utc)
+    )
+    updated_at = Column(
+        DateTime(timezone=True), default=lambda: datetime.now(timezone.utc)
+    )
+
+
+class HumanEvaluationScenarioDB(Base):
+    __tablename__ = "human_evaluations_scenarios"
+
+    id = Column(
+        UUID(as_uuid=True),
+        primary_key=True,
+        default=uuid.uuid7,
+        unique=True,
+        nullable=False,
+    )
+    user_id = Column(UUID(as_uuid=True), ForeignKey("users.id"))
+    user = relationship("UserDB")
+    evaluation_id = Column(UUID(as_uuid=True), ForeignKey("human_evaluations.id"))
+    evaluation = relationship("HumanEvaluationDB")
+    inputs = Column(JSONB)  # List of HumanEvaluationScenarioInput
+    outputs = Column(JSONB)  # List of HumanEvaluationScenarioOutput
+    vote = Column(String)
+    score = Column(JSONB)
```

Comment:
On `EvaluationDB`, the `score` column is currently an integer. Could you please specify the expected dictionary structure for this column, as it's defined as `JSONB`?

Reply:
we don't have a model for it. plain json
https://github.com/Agenta-AI/agenta/blob/main/agenta-backend/agenta_backend/models/db_models.py#L251